providers/openlineage/tests/unit/openlineage/extractors/test_manager.py

```diff
@@ -51,7 +51,7 @@
         # TODO: Remove once provider drops support for Airflow 2
         from airflow.utils.context import Context
 
-    from task_sdk.tests.conftest import MakeTIContextCallable
+    from conftest import MakeTIContextCallable
```

Comment:
This isnt importing the right thing anymore. Turns out this needed the "I can improt task_sdk" at the top level.

Given this is the only use of such an "outside of tasksdk test import" I think we should just re-implement that here instead.

Reply:
Should be fixed now - with a little cheating but :shrug: